ppcp-gateway/class-angelleye-paypal-ppcp-payment.php

```diff
@@ -4121,14 +4143,14 @@ public function angelleye_ppcp_add_payment_source($body_request, $order_id) {
             $order = wc_get_order($order_id);
             $user_id = (int) $order->get_customer_id();
             $all_payment_tokens = $this->angelleye_ppcp_get_all_payment_tokens_for_renewal($user_id);
-            $payment_tokens_id = get_post_meta($order_id, '_payment_tokens_id', true);
+            $payment_tokens_id = angelleye_ppcp_get_post_meta($order, '_payment_tokens_id', true);
             if (empty($payment_tokens_id)) {
-                $payment_tokens_id = get_post_meta($order_id, 'payment_token_id', true);
+                $payment_tokens_id = angelleye_ppcp_get_post_meta($order, 'payment_token_id', true);
```

Comment:
I am not sure why we're using 2 meta keys to store/retrieve the token ID? can you please clarify?

_payment_tokens_id & payment_tokens_id